.github/workflows/semver-checks.yml

```diff
@@ -37,3 +37,31 @@ jobs:
           message: |
             :rotating_light: API BREAKING CHANGE DETECTED
 
+  Feature:
+    name: Non additive cargo features - stable toolchain
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v4
+      - name: "Install Rustup"
+        uses: dtolnay/rust-toolchain@stable
+      - name: "Install cargo-binstall"
+        uses: cargo-bins/cargo-binstall@main
+      - name: "Binstall cargo-semver-checks"
+        run: cargo binstall cargo-semver-checks --no-confirm
```

Comment:
As discussed, we need to pin the version (and the Rust version). Sorry for not realizing this sooner.

Reply:
I think we need only to pin the rust version right? As per

> In your case, you're generating the rustdoc JSON with some version of Rust, and cargo-semver-checks will detect the format version and (if supported) adjust itself automatically, yes.
>
> @obi1kenobi

Second, this is a dynamic pin, right? Same as we do with `$(cat ./nightly-version)`?